### ISSUE TITLE
Version of firmware on GitHub

### ISSUE THREAD

**Issue by 20051971** [makerbase-mks/MKS-DLC32#459]
Hello everyone,

I just bought a DLC32 card with a TS35 screen.
The firmware installed on the card indicated by the GRBL console is:

[PRODUCER:MKS DLC32]
[V2.30(8M.H35.20231206)]
[VER:1.1h.2023120602:]
[OPT:VMPH,63,512]
Target buffer size found
[MSG:Using machine:MKS DLC32]
[MSG:No Wifi]

On the LCD screen:

Board:MKS DLC32 V2.1
Firmware:V2.30(8M.H35.20231206)

The firmware that seems to me to be the most recent on GitHub in "MKS-DLC32/MKS-DLC32-main/firmware/TS35/Laser/Normal /Board_V2.0/" is V2.30_H35_2022_1223_N_ZX_001.bin
The firmware on my card seems more recent than that published on GitHub. Can you confirm me that I do not need to update it.

Thank you in advance for your help.

Frederic from France.

**Comment by Kotdnz** [comment]
+1

**Comment by costycnc** [comment]
yes ... you are right ... also when arrive the wifi is off and when change to wifi on over serial cable ,after restart begin again wifi off ... so i prefer to reinstall the old firmware.Also some firmware have bugs ,for example one firmware that have inserted sdcard at begin can connect with wifi but web page cant be open ,another firmware is not in default as access point station to be see in wifi stations ... so i find this and i remain with this for now https://github.com/costycnc/mks-dlc32-refirmware
